app/assets/javascripts/spotlight/users.js

```diff
@@ -27,13 +27,15 @@
     }
 
     function clear_errors(element) {
-      element.find('.has-error').removeClass('has-error');
-      element.find('.help-block').remove(); // Remove the error messages
+      element.find('.has-error')
```

Comment:
Is this really equivalent to the old behavior? I'm not too clear on how jquery chaining works, I guess.